City-of-Helsinki/drupal-hdbt
# UHF-4768 unifying maps

# Unify tpr unit map & map paragraph [UHF-4768](https://helsinkisolutionoffice.atlassian.net/browse/UHF-4768)

Unified the styles and created one template that both the map paragraph and the tpr unit map uses.

## What was done
* Created a map component template that can be used in the paragraph as well as on the tpr unit page
* Unified the map styles and remove old unnecessary styles

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-4768-unifying-maps`
* Run `make drush-cr`

## How to test
* Add a map paragraph to a standard page
* Check that the styles work in desktop and in mobile
* Go to some tpr unit page and check that the map styles are the same there and that they also work on desktop and in mobile
